ios/Exponent/Kernel/DevSupport/EXKernelDevKeyCommands.m

```diff
@@ -122,6 +161,35 @@ + (void)initialize
   RCTSwapInstanceMethods([UIResponder class],
                          @selector(keyCommands),
                          @selector(EX_keyCommands));
+
+  SEL originalKeyboardSelector = NSSelectorFromString(@"handleKeyUIEvent:");
+  RCTSwapInstanceMethods([UIApplication class],
+                         originalKeyboardSelector,
+                         @selector(handleKeyUIEventSwizzle:));
+}
+
++(void)handleKeyboardEvent:(UIEvent *)event
+{
+  // NOTE: throttle the key handler because on iOS 9 the handleKeyCommand:
```

Comment:
we don't support iOS 9 🤔 do we really need that?

Reply:
I think this was included in a copy-paste from our other handler - it doesn't throttle anyways so I will remove